### PR TITLE
Update runner to load tsconfig from jest rootDir

### LINKED ISSUE #2: Doesn't respect tsconfig.json
I thought the TS Compiler API would do it by default, but looks like I need to make changes to support tsconfig options.

### PR DESCRIPTION
**Pull request by kwelch**
I attempted this a few time locally with node and linked to a separate project. 

Let me know if there is anything else you would like to see, but this should resolve #2

### CODE REVIEW

**Comment by kwelch** [comment]
Hey @azz, 

Is there anything else I can do to help get this approved and shipped?

Thanks, 
--Kyle

File: src/runTsc.js [hunk 1]
```diff
@@ -46,12 +47,31 @@ const convertErrors = ({ start, end, errors, testPath }) => ({
...
+    const settings = ts.convertCompilerOptionsFromJson(
+      config['compilerOptions'] || {},
+      process.cwd()
+    );
+
+    const options = Object.assign({}, { noEmit: true }, settings.options);
+
     const start = +new Date();
-    const program = ts.createProgram([testPath], {
-      noEmit: true,
```
Comment (azz) [inline src/runTsc.js]: Will removing this cause files to be written to disk?

**Comment by azz** [comment]
Hey, sorry I missed your PR, great stuff. 

I've been overrun by notifications since the last break I took, so it flew under the radar.

image

**Comment by azz** [comment]
:tada: https://github.com/azz/jest-runner-tsc/releases/tag/v1.2.0

**Comment by kwelch** [comment]
Awesome! Thanks. Didn't mean to push, I understand the overload of notification, just going for a bump.

**Pull request merged**